lib/ansible/cli/inventory.py

```diff
@@ -93,6 +93,8 @@ def parse(self):
         # graph
         self.parser.add_option("-y", "--yaml", action="store_true", default=False, dest='yaml',
                                help='Use YAML format instead of default JSON, ignored for --graph')
+        self.parser.add_option('--toml', action='store_true', default=False, dest='toml',
+                               help='Use TOML format instead of default JSON, ignored for --graph')
```

Comment:
These changes to add `--toml` to `ansible-inventory` aren't really necessary. Although I figured it would more easily enable someone to convert from ini to toml if we for some reason decided to make that push.

Reply:
im fine with this one, agreed that it is nice for those changing formats